SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslator/SpatialVirtualWorkerTranslatorTest.cpp

```diff
@@ -22,17 +22,6 @@ VIRTUALWORKERTRANSLATOR_TEST(Given_init_is_not_called_THEN_return_not_ready)
 	return true;
 }
 
```

Comment:
Is it intentional that this test is removed?

Reply:
Yes. The earlier implementation had a very naive way to decide if the translator was ready. Now we base it off of getting data over the wire, which isn't possible to test right now.